jetty-core/jetty-deploy/src/main/config/modules/deployment-scanner.mod

```diff
@@ -20,14 +20,15 @@ etc/jetty-deployment-scanner.xml
 
 [ini-template]
 #tag::documentation[]
-## The web application deploy directory name (relative to $JETTY_BASE).
-# jetty.deploy.monitoredDir=webapps
+## The web application deploy directory name (relative to $jetty.base)
```

Comment:
Restore `$JETTY_BASE`.

Reply:
Sorry, that was the result of a merge goof.